ui/src/app/services/conversion/conversion.service.ts

```diff
@@ -283,6 +284,39 @@ export class ConversionService {
     ]
   }
 
+  getCheckConstraints(tableId: string, data: IConv): ICcTabData[] {
+    let srcArr = data.SrcSchema[tableId].CheckConstraints || []
+    let spArr = data.SpSchema[tableId].CheckConstraints || []
+    let res: ICcTabData[] = []
+     if (srcArr.length > spArr.length) {
+      for (let i = 0; i < srcArr.length; i++) {
+        res.push({
+          srcSno: `${i + 1}`,
```

Comment:
Do we need to have src number here? this will cause issues if a Check constraint is deleted. We should just number a list